examples/spark/pytorch/pytorch_lightning_spark_mnist.py

```diff
@@ -184,6 +188,14 @@ def on_train_end(self, trainer, model):
                                    verbose=True,
                                    mode='max'))
 
+    if False:
+        prof_dir = tempfile.mkdtemp()
```

Comment:
Maybe we should just use `log_dir`?

Reply:
Yes, makes sense.  When we don't pass in a dirname, lightning will pick the dirname from the logger's log_path.